File: Codigo/Obligatorio/codigo/ArenaGestor/ArenaGestor.APIContracts/Concert/ConcertGetConcertsDto.cs
namespace ArenaGestor.APIContracts.Concert
{
    public class ConcertGetConcertsDto
    {
        public string TourName { get; set; }
        public ConcertGetDateRangeConcertsDto DateRange { get; set; }
        public bool Upcoming { get; set; }

    }
}

File: Codigo/Obligatorio/codigo/ArenaGestorFront/src/app/models/Concerts/ConcertGetConcertsDto.ts
import { ConcertGetDateRangeConcertsDto } from "./ConcertGetDateRangeConcertsDto";

export class ConcertGetConcertsDto {
    tourName: String = "";
    upcoming: Boolean = false;
    dateRange: ConcertGetDateRangeConcertsDto = new ConcertGetDateRangeConcertsDto();
}